src/main/kotlin/com/atlassian/performance/tools/report/api/judge/RelativePerformanceStabilityJudge.kt

```diff
@@ -42,4 +43,7 @@ class RelativePerformanceStabilityJudge {
             }
         )
     }
+
+    private fun extractDurationBasedOnLabel(dispersions: Map<String, Duration>, label: String) =
+        dispersions[label] ?: error("There is no $label")
```

Comment:
`RelativeTypicalPerformanceJudge` had a similar problem. We might want to fix it the same way. Apart from consistency, we'd also gain the cohort info in the error.

Reply:
hmmm in the `RelativeTypicalPerformanceJudge` there is code like that already.
```
val baselineCenter = baselineStats.locations[label]
            ?: return FailedAssertionJUnitReport(reportName, "No action $label results for $baselineCohort")
val experimentCenter = experimentStats.locations[label]
            ?: return FailedAssertionJUnitReport(reportName, "No action $label results for $experimentCohort")
```
I will update my message to match the convention.

Reply:
Sounds good